### DISCUSSION TITLE
Questions on image download buttons and italics

### DISCUSSION THREAD

**Discussion by geniza** [CollectionBuilder/collectionbuilder.github.io#77] [Q&A]
Dear CollectionBuilder,

I'm reaching the end of the development process for my project and have a few final questions:

1. How can I remove the download buttons from the display of collection items?
2. Also, when I'm including citations for specific collection entries I'm having difficulty placing the titles of books and journals in italics. What's the best way to code italics within the .csv or in Google Sheets before converting to .csv?

I've included images of the exact features that these questions cover. Thanks so much :) 

![Screen Shot 2023-01-30 at 2 09 58 PM](https://user-images.githubusercontent.com/106448975/215573248-3c4507d3-4af4-440b-bd75-1ae537643315.png)
![Screen Shot 2023-01-30 at 2 12 35 PM](https://user-images.githubusercontent.com/106448975/215573250-e2e3d73b-e3c8-4a8b-ba82-cc41243adc16.png)
![Screen Shot 2023-01-30 at 2 10 06 PM](https://user-images.githubusercontent.com/106448975/215573254-e24c1e46-c540-4940-83be-bd886d19309b.png)

**Comment by evanwill** [comment]
@geniza 
 
Q1: removing the download buttons will require a slight modification to the file "_includes/js/item-js.html" in your repository. 
In your project, open [item-js.html](https://github.com/geniza/paleographicalglossary/blob/main/_includes/js/item-js.html), scroll down to line 71, which looks like:

```
document.getElementById("item-link").innerHTML = itemLink;
```

Comment out this single line, by adding `//` in front, so it looks like:

```
// document.getElementById("item-link").innerHTML = itemLink;
```

Q2: adding italics can be done by using html inline markup directly in your metadata fields. Please note, any formatting you add using Sheet's visual formatting will not be reflected in the final CSV (i.e. making something bold, italic, highlighted, etc). Instead, you will have to add italics using the html tag `<em>`. The html tag goes around the text you want to apply the style to, e.g. `<em>italic text</em>`. 
For example, in your metadata field "citation", your first row has the value:

```
"►Ben-Naeh, Yaron. Jews in the Realm of the Sultans: Ottoman Jewish Society in the Seventeenth Century. Heidelberg: Mohr Siebeck, 2008.

►Pamuk, Şevket. A Monetary History of the Ottoman Empire. Cambridge: Cambridge University Press, 2000."
```

If you want the titles to be italic, you might mark this up like:

```
"►Ben-Naeh, Yaron. <em>Jews in the Realm of the Sultans: Ottoman Jewish Society in the Seventeenth Century</em>. Heidelberg: Mohr Siebeck, 2008.<br>
►Pamuk, Şevket. <em>A Monetary History of the Ottoman Empire</em>. Cambridge: Cambridge University Press, 2000."
```

The `<br>` tag provides a line break. 

It is also possible to format metadata using Markdown, but that requires some modification of the template. One option is to treat the "citation" field differently than the other metadata (in a separate box or something), and use Markdown to format the contents (e.g. italic would be like `*italic text*` )--if that seems helpful for your collection, let me know and I can open a PR with the modification.
